plugins/remotefortressreader/remotefortressreader.cpp

```diff
@@ -256,6 +255,252 @@ command_result dump_bp_mods(color_ostream &out, vector <string> & parameters)
+            remove(filename);
+            ofstream output;
+            output.open(filename);
+            output << "Pos: " << plant->pos.x << ", " << plant->pos.y << ", " << plant->pos.z << endl;
+            output << "Material: " << plant->material << endl;
+            output << "Grow Counter: " << plant->grow_counter << endl;
+            output << "Damage Flags: ";
+            if(plant->damage_flags.bits.is_burning)
+                output << "is_burning, ";
+            if(plant->damage_flags.bits.is_drowning)
+                output << "is_drowning, ";
+            if(plant->damage_flags.bits.anon_1)
+                output << "anon_1, ";
+            output << endl;
+            output << "HP: " << plant->hitpoints << endl;
+            output << "Update Order: " << plant->update_order << endl;
+            output << "Site ID: " << plant->site_id << endl;
+            output << "SRB ID: " << plant->srb_id << endl;
+            auto treeInfo = plant->tree_info;
+            output << "Dim X: " << treeInfo->dim_x << endl;
+            output << "Dim Y: " << treeInfo->dim_y << endl;
+            output << "Dim Z: " << treeInfo->body_height << endl;
+            for(int z = 0; z < treeInfo->body_height; z++)
+            {
+                for(int y = -1; y < treeInfo->dim_y; y++)
+                {
+                    for(int x = -1; x < treeInfo->dim_x; x++)
+                    {
+                        if(x < 0)
+                        {
+                            if(y < 0)
+                            {
+                                output << " ";
+                                continue;
+                            }
+                            output << y%9;
+                            continue;
+                        }
+                        if(y < 0)
+                        {
+                            output << x%9;
+                            continue;
+                        }
+                        auto tile = treeInfo->body[z][x + treeInfo->dim_x*y];
+                        if(tile.bits.blocked)
+                            output << "x";
+                        else if(tile.bits.twigs)
+                            output << "░";
+                        else if(tile.bits.branches)
+                        {
+                            if(tile.bits.connection_east)// East Connection
+                            {
+                                if(tile.bits.connection_south) // South Connection
+                                {
+                                    if(tile.bits.connection_west) // West Connection
+                                    {
+                                        if(tile.bits.connection_north) // North Connection
+                                            {
+                                                if(tile.bits.trunk)
+                                                    output << "╬";
```

Comment:
I would prefer escapes here (worried about this not working with some compilers or for some developers, although that might be unfounded)

Reply:
Solved it by getting rid of that function entirely.